libpod/container_internal.go

```diff
@@ -828,26 +828,24 @@ func (c *Container) mountStorage() (string, error) {
 		return c.state.Mountpoint, nil
 	}
 
-	if !rootless.IsRootless() {
-		// TODO: generalize this mount code so it will mount every mount in ctr.config.Mounts
-		mounted, err := mount.Mounted(c.config.ShmDir)
-		if err != nil {
-			return "", errors.Wrapf(err, "unable to determine if %q is mounted", c.config.ShmDir)
-		}
+	// TODO: generalize this mount code so it will mount every mount in ctr.config.Mounts
```

Comment:
Do we or should we have an issue for that?  If so could you note it here please?